CONTRIBUTING.md

```diff
@@ -1,178 +1,3 @@
-discussion evolves before implementation begins.
-
-### Translations
-
-You can also contribute to Openverse by translating it.
-
-An overview of Openverse translations is here:
-[https://translate.wordpress.org/projects/meta/openverse/](https://translate.wordpress.org/projects/meta/openverse/)
-
-A getting started guide for translating on GlotPress (the software behind
-[translate.wordpress.org](http://translate.wordpress.org)) is here:
-[https://make.wordpress.org/polyglots/handbook/translating/glotpress-translate-wordpress-org/#getting-started](https://make.wordpress.org/polyglots/handbook/translating/glotpress-translate-wordpress-org/#getting-started)
-
-### Providers
-
-Openverse is powered by upstream providers of openly licensed media. You can
-help expand Openverse by identifying sources of Creative Commons licensed
-media - we're always looking to broaden our dataset.
-
-[Our currently list of providers which have been identified but are not yet being ingested can be found here](https://github.com/WordPress/openverse/issues?q=is%3Aopen+is%3Aissue+label%3A%22%E2%98%81%EF%B8%8F+provider%3A+any%22%2C%22%E2%98%81%EF%B8%8F+provider%3A+audio%22%2C%22%E2%98%81%EF%B8%8F+provider%3A+images%22%2C%22%E2%98%81%EF%B8%8F+provider%3A+text%22%2C%22%E2%98%81%EF%B8%8F+provider%3A+video%22%2C%22%E2%98%81%EF%B8%8F+provider%3A+3D+models%22).
-
-You can use the
-[New Source Suggestion for Openverse](https://github.com/WordPress/openverse/issues/new?assignees=&labels=%F0%9F%9A%A6+status%3A+awaiting+triage%2C%F0%9F%A7%B9+status%3A+ticket+work+required%2C%E2%98%81%EF%B8%8F+provider%3A+any&template=new_source_suggestion.yml&title=%3CSource+name+here%3E)
-issue template to submit whatever sources you find.
-
-### Feedback
-
-Feedback like reporting bugs and missing features is big help for Openverse. We
-want Openverse to be a bug-free app with great user experience and providing
-feedback is a way to put any shortcomings on our radar and get them resolved.
-
-You can report bugs, request features and see all other forms of feedback we
-would like to receive by
-[submitting a new issue](https://github.com/WordPress/openverse/issues/new/choose)
-on GitHub. All are welcome to write issues and the Openverse maintainers have
-deep gratitude for those who do. Note that you will need a GitHub account to
-create new issues.
-
-You can also provide feedback via any of our other
-[communication channels](README.md#keep-in-touch).
-
-### Referrals
-
-If you know folks who have expertise in any of the above areas who you think
-might be interested in contributing to open source, send them our way! We're
-happy to help onboard folks to the project itself, as well as the tools and
-technologies we use.
+```{note}
+**Note:** This contributing guide has been moved to [docs.openverse.org/general/contributing](https://docs.openverse.org/general/contributing). Please refer to the new location for the latest version.
+```
```

Comment:
The syntax for notes in GitHub is different compared to Sphinx. Here it should be:
```suggestion
> **Note**
> This contributing guide has been moved to [docs.openverse.org/general/contributing](https://docs.openverse.org/general/contributing). Please refer to the new location for the latest version.
```
```